bin/internal/update_dart_sdk.sh

```diff
@@ -59,6 +59,8 @@ if [ ! -f "$ENGINE_STAMP" ] || [ "$ENGINE_VERSION" != `cat "$ENGINE_STAMP"` ]; t
       exit 1
       ;;
   esac
+  
+  FIND=/usr/bin/find
```

Comment:
We would like this to check for existence of this file, and if it is not there, fall back on the old behavior. Thanks!

Reply:
OK, I have just updated the branch for the fall back logic, please take a look when you have time. 👀